conf/machine/genericx86-64.extra.conf

```diff
@@ -37,7 +37,7 @@ WICVARS:append = " OMNECT_GRUBENV_FILE"
 OMNECT_GRUBENV_FILE = "${LAYERDIR_omnect}/files/grubenv"
 
 # adu compatibility id
-OMNECT_ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "0"
+OMNECT_ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "1"
```

Comment:
not sure this is really a problem for adu ... the adu update doesn't touch the factory partition and other partition sizes didn't change. i assume the adu update works seamlessly.

Reply:
good point; think you are right, this is not a breaking change... will test with adu

Reply:
test with adu was successfully